handlers/discount-api/test/getNextPaymentDate.test.ts

```diff
@@ -0,0 +1,65 @@
+import { getNextNonFreePaymentDate } from '@modules/zuora/billingPreview';
+
+test('getNextPaymentDate fails if there are no payments in the preview', () => {
```

Comment:
Would the description be more accurate?

```suggestion
test('getNextPaymentDate fails if there are no non-free payments in the preview', () => {
```

Reply:
better still, flip the language completely
`getNextNonFreePaymentDate fails if all payments are free`